### PR TITLE
[Build] Include the HUD "package" with the CI and release process

### PR DESCRIPTION
**Pull request by philliphoff**
Adds support for packaging and releasing the HUD "package" in the same manner as the Client itself.

### CODE REVIEW

File: _temp/hud/package.json [hunk 1]
```diff
@@ -26,21 +26,23 @@
...
+    "gulp-zip": "^3.2.0",
     "imports-loader": "0.6.5",
     "jscs": "2.3.4",
     "jshint": "2.8.0",
     "jshint-loader": "0.8.3",
     "lodash": "^3.10.1",
     "minimist": "1.2.0",
     "moment": "^2.10.6",
-    "node-sass": "3.3.3",
+    "node-sass": "3.4.2",
```
Comment (avanderhoorn) [inline _temp/hud/package.json]: I remember that I had issues with updating to later versions of `node-sass`, but I don't know if that was for just the client or if I had the problem with HUD as well... do you know if things still work ok?

**Comment by avanderhoorn** [comment]
One minor question about sass dependency update, but besides that, looks good :shipit:

File: _temp/hud/package.json (hunk 1 above)
Comment (philliphoff) [inline _temp/hud/package.json]: Travis didn't like the current version so I bumped it to the same version used by the client, so I wouldn't think there would be a problem.  If I pull in a newly-built version of the HUD into the Server, things still seem to work as expected.

Comment (avanderhoorn) [inline _temp/hud/package.json]: Great!

**Pull request merged**